src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -104,6 +104,7 @@ public void TrackEvent(string eventName, IDictionary<string, string> properties
 
         /// <summary>
         /// Send an <see cref="EventTelemetry"/> for display in Diagnostic Search and aggregation in Metrics Explorer.
+        /// Create a separate EventTelemetry instance for each call to TrackEvent.
```

Comment:
Can you please use <see cref="Microsoft.ApplicationInsights.TelemetryClient.TrackEvent(Microsoft.ApplicationInsights.Extensibility.DataContracts.TelemetryEvent)"/>